content/zh/news/releases/1.13.x/announcing-1.13/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 发布 Istio 1.13
+linktitle: 1.13
+subtitle: 重大更新
+description: Istio 1.13 发布公告。
+publishdate: 2022-02-11
+release: 1.13.0
+skip_list: true
+aliases:
+    - /news/announcing-1.13
+    - /news/announcing-1.13.0
+---
+
+我们很高兴地宣布 Istio 1.13 即将发布！
+
+{{< relnote >}}
+
+这是 2022 年 Istio 的第一个版本发布。我们要感谢整个 Istio 社区为发布 Istio 1.13.0 所做出的贡献。特别感谢发布经理 Steven Landow（Google），Lei Tang（Google）和 Elizabeth Avelar（SAP），以及测试与发布工作组负责人 Eric Van Norman（IBM）提供的帮助和指导。
+
+{{< tip >}}
+Istio 1.13.0 官方支持 Kubernetes 版本为 `1.20` 至 `1.23`。
```

Comment:
```suggestion
Istio 1.13.0 正式支持的 Kubernetes 版本为 `1.20` 至 `1.23`。
```

Reply:
Fix all the issues, thank you for your knowledge.